packages/react-native/src/Authenticator/hooks/useFieldValues/utils.ts

```diff
@@ -130,6 +130,7 @@ export const getTypedField = ({
   ...field
 }: AuthenticatorLegacyField): TypedField => {
   const type = getFieldType(machineFieldType);
+  const testID = `amplify__text-field__input-${name}`;
```

Comment:
"amplify" might be a bit too generic here IMO. Maybe replace with "authenticator"